static/app/views/releases/detail/overview/releaseComparisonChart/index.tsx

```diff
@@ -201,6 +207,66 @@ function ReleaseComparisonChart({
       ? releaseCrashFreeSessions - allCrashFreeSessions
       : null;
 
+  const releaseHealthySessions = getSessionStatusRate(
```

Comment:
oh wow this file is very long 😮

Reply:
Right now it is. I'll be adding conditions to what charts you have access (performance, discover, sessions) and that's when I plan to break it down 👍